user/serializers.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
```

Comment:
Including the `password` field in the serializer's fields is correct, but ensure that it is handled securely. The `extra_kwargs` already specify `write_only` for the password, which is good practice.